includes/admin/tools/views/html-admin-page-data.php

```diff
@@ -56,7 +56,8 @@
 						$args = array(
 							'name'   => 'form_id',
 							'number' => - 1,
-							'chosen' => true,
+							'chosen' => false,
```

Comment:
@raftaar1191 why are you disabling `chosen`?

Reply:
Because after the ajax complete we need to reset the `Form` to it's default value and chosen was creating some issues. So I thought to remove it 

Without chosen parameter:
![image](https://user-images.githubusercontent.com/22215595/32747255-33303554-c8de-11e7-9d4c-148f736318a2.png)

With Chosen parameter:
![image](https://user-images.githubusercontent.com/22215595/32747288-4cd04300-c8de-11e7-9689-a02f7540ecb3.png)

@DevinWalker Have a look at the images I don't think it making a much more diff. If you want I can again add that

Reply:
@raftaar1191 I'd rather keep it since if you have a lot of forms it's helpful. As well, it should use AJAX to lookup forms not present in the dropdown options. There's a param for that.

The first option should actually say 

> Select Form

Not plural "forms" since you can only select one.

Reply:
Okay @DevinWalker adding that

Oop I did not see that `Select Forms` chaining that

Reply:
@DevinWalker Code Updated

Screensort :
![image](https://user-images.githubusercontent.com/22215595/32748502-7b53f9a2-c8e2-11e7-8983-6a21fe747f9c.png)